SECURITY-TESTING-GUIDE.md

```
@@ -0,0 +1,452 @@
+| Security Headers | 1 test | 100% |
+
+**Overall Expected:** 26/26 tests passing (100%)
+
+---
+
+## Reporting Security Issues
+
+If you find a security vulnerability during testing:
+
+1. **DO NOT** open a public GitHub issue
+2. Email security@invoiceplane.com with:
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)